README.md

```diff
@@ -16,7 +18,7 @@ The wallpaper may be changed randomly or a custom image can be defined in the Sk
 * Mycroft AI
 
 ## Supported Devices 
-platform_mark2 
+Mark II 
```

Comment:
I think this needs to be `platform_mark2` for Selene to display the icon. Is that right?

Reply:
You are probably right.  Didn't even think about that.  It just looks weird in a README.  We should probably change it at some point.